integration-tests/ecs/src/ecs-fargate-elb/ecs-fargate-elb.ts

```diff
@@ -1,89 +1,115 @@
-import { copyFileSync } from 'fs';
-import { dirname, join } from 'path';
-import { Duration, Stack, StackProps } from 'aws-cdk-lib';
-import { SubnetType, Vpc } from 'aws-cdk-lib/aws-ec2';
-import { Platform } from 'aws-cdk-lib/aws-ecr-assets';
-import { AwsLogDriver, Cluster, ContainerImage, FargateTaskDefinition, Protocol as EcsProtocol, Secret as EcsSecret } from 'aws-cdk-lib/aws-ecs';
-import { ApplicationLoadBalancedFargateService } from 'aws-cdk-lib/aws-ecs-patterns';
-import { Secret } from 'aws-cdk-lib/aws-secretsmanager';
-import { Construct } from 'constructs';
-import { ApplicationProtocol, Protocol } from 'aws-cdk-lib/aws-elasticloadbalancingv2';
+import {copyFileSync, existsSync} from 'fs';
+import {dirname, join} from 'path';
+import {Duration, Stack, StackProps} from 'aws-cdk-lib';
+import {IpAddresses, SubnetType, Vpc} from 'aws-cdk-lib/aws-ec2';
+import {Platform} from 'aws-cdk-lib/aws-ecr-assets';
+import {
+  AwsLogDriver,
+  Cluster,
+  ContainerImage,
+  FargateTaskDefinition,
+  Protocol as EcsProtocol,
+  Secret as EcsSecret
+} from 'aws-cdk-lib/aws-ecs';
+import {ApplicationLoadBalancedFargateService} from 'aws-cdk-lib/aws-ecs-patterns';
+import {Secret} from 'aws-cdk-lib/aws-secretsmanager';
+import {Construct} from 'constructs';
+import {ApplicationProtocol, Protocol} from 'aws-cdk-lib/aws-elasticloadbalancingv2';
+import {LogGroup} from "aws-cdk-lib/aws-logs";
 
 export class EcsFargateElbStack extends Stack {
 
-    constructor(scope: Construct, id: string, props: StackProps) {
-        super(scope, id, props);
+  constructor(scope: Construct, id: string, props: StackProps) {
+    super(scope, id, props);
 
-        const lumigoTokenSecret = EcsSecret.fromSecretsManager(Secret.fromSecretNameV2(this, 'Secret', 'AccessKeys'), 'LumigoToken');
+    const suffix = props.tags?.['lumigo:suffix'] || '-dev';
 
-        const vpc = new Vpc(this, 'EcsFargateElbVpc', {
-            vpcName: 'EcsFargateElb',
-            cidr: '10.0.0.0/16',
-            maxAzs: 3, // Default is all AZs in region
-            natGateways: 1,
-            subnetConfiguration: [
-                {
-                    name: 'private-subnet',
-                    subnetType: SubnetType.PRIVATE_WITH_EGRESS,
-                    cidrMask: 24,
-                },
-                {
-                    name: 'public-subnet',
-                    subnetType: SubnetType.PUBLIC,
-                    cidrMask: 24,
-                },
-            ],
-        });
+    const lumigoTokenSecret = EcsSecret.fromSecretsManager(
```

Comment:
We should use https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.SecretValue.html#static-plainwbrtextsecretspan-classapi-icon-api-icon-deprecated-titlethis-api-element-is-deprecated-its-use-is-not-recommended%EF%B8%8Fspan instead with the LumigoToken read from process.env